java-squid/src/main/java/org/sonar/java/ast/JavaAstScanner.java

```diff
@@ -69,16 +55,10 @@ public JavaAstScanner(ActionParser<Tree> parser) {
    */
   public JavaAstScanner(JavaAstScanner astScanner) {
     this.parser = astScanner.parser;
-    this.index = astScanner.index;
   }
 
   public void scan(Iterable<File> files) {
```

Comment:
Given the result of that method I guess that either this one or simpleScan should go away...